### PR TITLE
[WIP] Updates of the builder backend

### PR DESCRIPTION
**Pull request by denji**
Updates of the builder backend
====

sic

Cleanup unchanged, moved all external dependencies on the modules:
* `package.json`
* `./public/js/[module].{css,js,es6,jsx}`

`./assets/` folder left some folders which were changed by deploy stage.

Code style
---
* [ ] [`standard-js`](https://github.com/feross/standard)
* [ ] [`stylefmt`](https://github.com/morishitter/stylefmt)

Build
---
* [ ] Webpack 2.2
   * [x] webpack.config.js

### CODE REVIEW

**Comment by tboerger** [comment]
This needs to be tested if it breaks anything.

**Comment by denji** [comment]
In the future is to make bootstraping gulp/webpack2/rollup without the use of storing unnecessary unpackaged files.

**Comment by tboerger** [comment]
Yeah I would like to add webpack or something like that as well, but the asset bundle will always be part of the repo because we don't want to blow up the build requirements.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/modules/public/bindata.go
```
Comment (tboerger) [inline .gitignore]: Please revert these changes, there is so much junk inside that shouldn't be part of a project gitignore. Build your own global gitignore that ignores things like that.

**Comment by tboerger** [comment]
And the diff is grown so much that it's not really reviewable anymore -.-

**Comment by tboerger** [comment]
And one more nitpicking, please write meaningful commit messages :)

**Comment by tboerger** [comment]
After thinking about it, we should delay that to 1.1 because there is so much breaking potential

**Comment by denji** [comment]
I also think that is a v1.1+, changes in the global and possibly v1.2 :)

**Comment by bkcsoft** [comment]
You can't just "Update" PDF.js, since I had to remove a lot of stuff from it so as to not bloat the feature...

**Comment by bkcsoft** [comment]
@denji Do you actually have a reason to update all these dependencies?

You can't really expect anyone to review 1600 files... even if it's just deps...

**Comment by denji** [comment]
@bkcsoft Let it be, then I'll do squash & rebase.

**Comment by lunny** [comment]
I think this PR is so big that almost nobody could review that. So I close this one.
